react/src/core-components/TextCopyField/TextCopyField.module.css

```diff
@@ -7,7 +7,7 @@
   transition: color var(--transition-duration),
     background-color var(--transition-duration);
 
-  composes: c-button--secondary from '@tacc/core-styles/src/lib/_imports/components/c-button.css';
+  composes: c-button--secondary from '@tacc/core-styles/dist/components/c-button.css';
```

Comment:
@ Wes  , @tjgrafft , @sophia-massie  This was causing the errors we were seeing; so should make its way back to TUP and/or the future core-components package:

```
GET http://localhost:4200/tools/x-truncate.css net::ERR_ABORTED 404 (Not Found)
GET http://localhost:4200/objects/o-section.selectors.css net::ERR_ABORTED 404 (Not Found)
GET http://localhost:4200/tools/x-link.css net::ERR_ABORTED 404 (Not Found)
GET http://localhost:4200/c-button.selectors.css net::ERR_ABORTED 404 (Not Found)
```

Reply:
Thanks, yes. Bug in TUP. _The `/dist/` is preferred unless one need do something advanced._